core/src/test/java/zipkin2/reporter/AwaitableCallbackTest.java

```diff
@@ -23,19 +23,14 @@
 
 class AwaitableCallbackTest {
   @Test void awaitIsUninterruptable() {
-    AtomicBoolean returned = new AtomicBoolean();
     AwaitableCallback captor = new AwaitableCallback();
-    Thread thread = new Thread(() -> {
-      captor.await();
-      returned.set(true);
-    });
+    Thread thread = new Thread(captor::await);
     thread.start();
     thread.interrupt();
 
     assertThat(thread.isInterrupted()).isTrue();
     // The callback thread receiving an interrupt has nothing to do with the caller of the captor
     assertThat(Thread.currentThread().isInterrupted()).isFalse();
-    assertThat(returned.get()).isFalse();
```

Comment:
this was winky depending on JDK. The important thing is that we interrupted the thread launched and not the current one, so I removed this part